src/js/Meister.js

```diff
@@ -161,6 +161,27 @@ class Meister {
     }
 
     /**
+     * Returns filtered and formatted list of all registered plugins and their version.
+     *
+     * @return {string} List with plugins and their version.
+     */
+    static get versions() {
```

Comment:
I would suggest:

```
const registeredPlugins = PluginLoader.getRegistered();

        return registeredPlugins.map(plugin => {
            return {
                pluginName: plugin.name,
                pluginVersion: plugin.Plugin.pluginVersion,
            };
        });
```

This would show something like this:

<img width="450" alt="screen shot 2017-08-08 at 16 05 15" src="https://user-images.githubusercontent.com/5691892/29075672-671f6380-7c53-11e7-8d5f-b1158f02eb0e.png">